numpy/core/arrayprint.py

```diff
@@ -87,6 +87,9 @@ def _make_options_dict(precision=None, threshold=None, edgeitems=None,
     if sign not in [None, '-', '+', ' ']:
         raise ValueError("sign option must be one of ' ', '+', or '-'")
 
+    if legacy not in [None, '', '1.13']:
+        raise ValueError("legacy option can currently only be '1.13' or ''")
```

Comment:
I think this should warn, not error - otherwise, code which does not want 1.15 behavior (passing `legacy='1.14'`), will crash when run on 1.13.

Reply:
Agree.